mix.exs

```diff
@@ -14,6 +14,8 @@ defmodule Nerves.Runtime.MixProject do
       compilers: [:elixir_make | Mix.compilers()],
       make_targets: ["all"],
       make_clean: ["clean"],
+      make_error_message:
+        "nerves_runtime requires libmnl. It can be installed by running apt install libmnl-dev",
```

Comment:
Thanks, I think this is a really good idea and that it will help a lot of people who are thrown off by the default error message.

How about changing it to the following? Instead of just accepting this change blindly, could you formatting it across multiple lines using `"""`?

```suggestion
        "If the error message above says that libmnl.h can't be found, then the fix is to install libmnl. For example, run `apt install libmnl-dev` on Debian-based systems.",
```

Also, could you update the commit message to "Suggest installing libmnl on C compilation errors".